src/config/AppConfig.ts

```diff
@@ -60,6 +65,10 @@ const defaultAppConfig: AppConfig = {
     ],
     marketServerUrl: 'http://app.nemcn.io',
     articlesFeedUrl: 'https://symbol.github.io/symbol-rss-feeds/',
+    statisticServiceUrls: [
```

Comment:
Rather than Appconfig.ts
Maybe you should move to `src/config/NetworkConfig.ts`.
I saw some config there `defaultMainnetNetworkConfig` and `defaultTestnetNetworkConfig`

Reply:
+ hardcoded nodes in NetworkConfig are not removed.

Reply:
@cryptoBeliever for now, they are just used for testing and harvesting.

Reply:
Are you sure about harvesting? It should take nodes from /node/peers :thinking: 
For what is used in tests?

Reply:
yes the nodes are fetched from node/peers and we're merging the ngl static nodes to the list. https://github.com/symbol/desktop-wallet/blob/bba0a5efdfc64686b60311cc492823a1b33e65f6/src/components/NetworkNodeSelector/NetworkNodeSelectorTs.ts?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L147

Reply:
There should be no hard-coded nodes used. Those nodes will be eventually shut down and we don't want dead nodes on the harvesting list.

Reply:
I understand but I also believe we agreed on keeping harvesting non-changed and handling it later in a separate PR.

Reply:
Yes and not :) 
Not using statistic service and keep `/node/peers` on harvesting view is not a problem for this release.
Using the hardcoded list of nodes is a problem and we should remove it now. Will be big change to remove it now? If not please do.

Reply:
what happens to users who already have their keys linked to one of our static nodes?

Reply:
@bassemmagdy there will be no issue if you will remove this hardcoded list. Currently all NGL nodes have disabled harvesting anyway, so if user picked before NGL node his harvesting stopped working weeks ago and he have to choose another one.

Please remove line: ;)
https://github.com/symbol/desktop-wallet/blob/bba0a5efdfc64686b60311cc492823a1b33e65f6/src/components/NetworkNodeSelector/NetworkNodeSelectorTs.ts?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L147

Reply:
And if possible also remove entirely list of hardcoded nodes from production code.